lib/matplotlib/backends/backend_pgf.py

```diff
@@ -94,9 +94,15 @@ def repl_mathdefault(m):
 
 
 def common_texification(text):
-    """
+    r"""
     Do some necessary and/or useful substitutions for texts to be included in
     LaTeX documents.
+
+    This distinguishes text-mode and math-mode by splitting on the math
```

Comment:
'splitting' here confused me, because I thought the actual string would be split up. How about instead

"by replacing the math separator with ``\(\displaystyle %s\)``. Escaped math separators (``\$``) are ignored.